install/project/includes/configuration/configuration.inc.sample.php

```diff
@@ -165,9 +166,11 @@
 			define ('__APP_CSS_ASSETS__', __PROJECT_ASSETS__ . '/css');
 			define ('__APP_IMAGE_ASSETS__', __PROJECT_ASSETS__ . '/images');
 			define ('__APP_PHP_ASSETS__', __PROJECT_ASSETS__ . '/php');
+			define ('__APP_IMAGE_CACHE__', __APP_IMAGE_ASSETS__ . '/cache');
+			define ('__APP_UPLOAD__', __DOCROOT__ . __PROJECT_ASSETS__ . '/upload');
```

Comment:
I am concerned about security. Anything that can be uploaded by the user should be treated carefully, and putting it in the project/assets directory is not careful. However, we are uploading things that need to be directly accessed by the browser, so the TMP directory is not a good place for that. Not sure how to handle that.